src/metabase/driver/query_processor.clj

```diff
@@ -1,6 +1,9 @@
 (ns metabase.driver.query-processor
   "Preprocessor that does simple transformations to all incoming queries, simplifing the driver-specific implementations."
-  (:require (clojure [pprint :as pprint]
+  (:require [clj-time.core :as t]
+            [clj-time.coerce :as tc]
+            [clj-time.format :as tf]
```

Comment:
Could you keep the `clj-time` imports after `clojure` imports? And keep them sorted too? (`coerce`, `core`, `format`)? 

I'd also prefer grouping them like `(clj-time ...)` but that's a more of a personal preference.